2025-03-23 18:21:49,854 - INFO - Start polling
2025-03-23 18:21:50,116 - INFO - Run polling for bot @halal_cafe_usa_bot id=7805627856 - 'UZBEK USA HALAL FOOD \U0001f372'
2025-03-23 18:22:09,929 - INFO - Update id=48807923 is not handled. Duration 0 ms by bot id=7805627856
2025-03-23 18:23:03,712 - INFO - Polling stopped for bot @halal_cafe_usa_bot id=7805627856 - 'UZBEK USA HALAL FOOD \U0001f372'
2025-03-23 18:23:03,712 - INFO - Polling stopped
2025-03-23 18:23:16,368 - INFO - Start polling
2025-03-23 18:23:16,638 - INFO - Run polling for bot @halal_cafe_usa_bot id=7805627856 - 'UZBEK USA HALAL FOOD \U0001f372'
2025-03-23 18:23:21,674 - INFO - Update id=48807924 is not handled. Duration 0 ms by bot id=7805627856
2025-03-23 18:23:21,675 - ERROR - Cause exception while process update id=48807924 by bot id=7805627856
NameError: name 'restaurants_data' is not defined
Traceback (most recent call last):
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\middlewares\error.py", line 25, in __call__
    return await handler(event, data)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\middlewares\user_context.py", line 56, in __call__
    return await handler(event, data)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\fsm\middleware.py", line 42, in __call__
    return await handler(event, data)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\event\telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\event\handler.py", line 43, in call
    return await wrapped()
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 141, in _wrapped
    return await self._propagate_event(
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 174, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 141, in _wrapped
    return await self._propagate_event(
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\event\telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\event\handler.py", line 43, in call
    return await wrapped()
  File "c:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\main.py", line 100, in process_state
    if state not in restaurants_data:
NameError: name 'restaurants_data' is not defined
2025-03-23 18:23:30,153 - INFO - Update id=48807925 is not handled. Duration 0 ms by bot id=7805627856
2025-03-23 18:23:30,153 - ERROR - Cause exception while process update id=48807925 by bot id=7805627856
NameError: name 'restaurants_data' is not defined
Traceback (most recent call last):
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\middlewares\error.py", line 25, in __call__
    return await handler(event, data)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\middlewares\user_context.py", line 56, in __call__
    return await handler(event, data)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\fsm\middleware.py", line 42, in __call__
    return await handler(event, data)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\event\telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\event\handler.py", line 43, in call
    return await wrapped()
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 141, in _wrapped
    return await self._propagate_event(
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 174, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 141, in _wrapped
    return await self._propagate_event(
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\event\telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "C:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\env\lib\site-packages\aiogram\dispatcher\event\handler.py", line 43, in call
    return await wrapped()
  File "c:\Users\Amiri\Desktop\Find-Halal-Cafe-USA\main.py", line 100, in process_state
    if state not in restaurants_data:
NameError: name 'restaurants_data' is not defined
2025-03-23 18:33:32,240 - INFO - Polling stopped for bot @halal_cafe_usa_bot id=7805627856 - 'UZBEK USA HALAL FOOD \U0001f372'
2025-03-23 18:33:32,240 - INFO - Polling stopped
2025-03-23 18:36:24,990 - INFO - Start polling
2025-03-23 18:36:25,325 - INFO - Run polling for bot @halal_cafe_usa_bot id=7805627856 - 'UZBEK USA HALAL FOOD \U0001f372'
2025-03-23 18:36:33,781 - INFO - Update id=48807926 is handled. Duration 391 ms by bot id=7805627856
2025-03-23 18:36:38,755 - INFO - Update id=48807927 is handled. Duration 202 ms by bot id=7805627856
2025-03-23 18:38:46,722 - INFO - Update id=48807928 is handled. Duration 344 ms by bot id=7805627856
2025-03-23 18:40:15,675 - INFO - Update id=48807929 is handled. Duration 172 ms by bot id=7805627856
2025-03-23 18:40:18,431 - INFO - Update id=48807930 is handled. Duration 297 ms by bot id=7805627856
2025-03-23 18:40:21,393 - INFO - Update id=48807931 is handled. Duration 156 ms by bot id=7805627856
2025-03-23 18:41:21,961 - INFO - Polling stopped for bot @halal_cafe_usa_bot id=7805627856 - 'UZBEK USA HALAL FOOD \U0001f372'
2025-03-23 18:41:21,961 - INFO - Polling stopped
2025-03-23 18:41:28,245 - INFO - Start polling
2025-03-23 18:41:28,550 - INFO - Run polling for bot @halal_cafe_usa_bot id=7805627856 - 'UZBEK USA HALAL FOOD \U0001f372'
2025-03-23 18:41:35,798 - ERROR - \u274c Ошибка при проверке подписки 5810325088: Telegram server says - Bad Request: chat not found
2025-03-23 18:41:36,018 - INFO - Update id=48807932 is handled. Duration 468 ms by bot id=7805627856
2025-03-23 18:41:38,970 - ERROR - \u274c Ошибка при проверке подписки 5810325088: Telegram server says - Bad Request: chat not found
2025-03-23 18:41:39,067 - INFO - Update id=48807933 is handled. Duration 266 ms by bot id=7805627856
2025-03-23 18:44:29,740 - INFO - Polling stopped for bot @halal_cafe_usa_bot id=7805627856 - 'UZBEK USA HALAL FOOD \U0001f372'
2025-03-23 18:44:29,740 - INFO - Polling stopped
2025-03-23 18:53:52,871 - INFO - Start polling
2025-03-23 18:53:53,108 - INFO - Run polling for bot @halal_cafe_usa_bot id=7805627856 - 'UZBEK USA HALAL FOOD \U0001f372'
2025-03-23 18:54:04,978 - INFO - Update id=48807938 is handled. Duration 547 ms by bot id=7805627856
2025-03-23 18:54:23,030 - INFO - Update id=48807939 is handled. Duration 187 ms by bot id=7805627856
2025-03-23 18:54:34,676 - INFO - Update id=48807940 is handled. Duration 297 ms by bot id=7805627856
2025-03-23 18:54:41,421 - INFO - Update id=48807941 is handled. Duration 375 ms by bot id=7805627856
2025-03-23 18:54:45,875 - INFO - Update id=48807942 is handled. Duration 108 ms by bot id=7805627856
2025-03-23 18:54:47,895 - INFO - Update id=48807943 is handled. Duration 108 ms by bot id=7805627856
2025-03-23 18:54:48,819 - INFO - Update id=48807944 is handled. Duration 125 ms by bot id=7805627856
2025-03-23 18:55:59,675 - INFO - Polling stopped for bot @halal_cafe_usa_bot id=7805627856 - 'UZBEK USA HALAL FOOD \U0001f372'
2025-03-23 18:55:59,675 - INFO - Polling stopped
